packages/table/src/Table.tsx

```diff
@@ -35,19 +76,77 @@ export default function Table<Shape>({
   className,
   ...rest
 }: TableProps<Shape>) {
+  const [scrollState, setScrollState] = React.useState<ScrollState>(
+    ScrollState.None,
+  );
+  const divRef = React.useRef<HTMLDivElement>(null);
+  const viewportSize = useViewportSize();
+  console.log(viewportSize);
+
+  useIsomorphicLayoutEffect(() => {
+    const ref = divRef.current;
```

Comment:
can clear up the following blocks by early returning if `ref == null`

Reply:
👍